include/faker-cxx/System.h

```diff
@@ -0,0 +1,214 @@
+#pragma once
+
+#include <algorithm>
+#include <ctime>
+#include <random>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "../src/common/StringHelper.h"
```

Comment:
Please move all needed types in header file to include/types directory, move only types not all the declared variables from those files.